.circleci/config.yml

```diff
@@ -40,7 +40,9 @@ jobs:
       - run:
           name: Run lint check
           command: |
+            # NOTE(willkg): Use -ci services since they doesn't mount . as a volume
             docker-compose run linting-ci lint
+            docker-compose run frontend-ci lint
```

Comment:
We should do all the linting in this step with containers built via our Docker instructions.